src/licensedcode/data/rules/not-a-license_429.RULE

```diff
@@ -0,0 +1 @@
+label	= "GPL2"
```

Comment:
Since this is very close to a possible license tag of sorts, could there be a word before or after that could be included to make this more specific?

Reply:
Yes! Adding a line before.